stripe/src/main/java/com/stripe/android/view/StripeEditText.java

```diff
@@ -7,24 +7,24 @@
 import android.support.annotation.ColorInt;
 import android.support.annotation.ColorRes;
 import android.support.annotation.Nullable;
+import android.support.v7.widget.AppCompatEditText;
 import android.util.AttributeSet;
 import android.view.KeyEvent;
 import android.view.View;
 import android.view.inputmethod.EditorInfo;
 import android.view.inputmethod.InputConnection;
 import android.view.inputmethod.InputConnectionWrapper;
-import android.widget.EditText;
 
 import com.stripe.android.R;
 
 /**
- * Extension of {@link EditText} that listens for users pressing the delete key when there is
+ * Extension of {@link AppCompatEditText} that listens for users pressing the delete key when there is
```

Comment:
Lint was actually now throwing an Error, not just a warning, since I was extending EditText instead of AppCompatEditText. It doesn't hurt anything to use the appcompat libraries, so... make lint happy?